Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -296,10 +301,11 @@ public void computedCompletion(@NotNull final String completionId,
                                    @NotNull final List<IncludedSuggestionSet> includedSuggestionSets,
                                    @NotNull final List<String> includedElementKinds,
                                    @NotNull final List<IncludedSuggestionRelevanceTag> includedSuggestionRelevanceTags,
-                                   final boolean isLast) {
+                                   final boolean isLast,
+                                   String libraryFile) {
```

Comment:
We try to use `@Nullable`/`@NotNull` where possible for arguments and return types. Could you add `@Nullable` here and everywhere where this value is propagated?

Reply:
Okay I think I hit all of them!